2020-an-environment-for-sustainable-research-software-in-germany-and-beyond/manuscript.md

```diff
@@ -0,0 +1,2021 @@
+“virtual hardware” such as webspace, versioning systems, task-managing
+systems, and compute cycles. These demands can hardly be met without
+third-party funding (Kuchinke et al. 2016; Chang, Mills, and Newhouse
+2007; Aartsen et al. 2018; Gabella, Durinx, and Appel 2018).
+
+Recommendation: Creation of Adequate Funding Schemes
+----------------------------------------------------
+
+Funding is a crucial factor for sustaining research software. Currently
+available sources and instruments are not adequately shaped for the
+challenges and solutions outlined above. We recommend actions on the
+individual, organizational, and national level.
+
+Existing project-focused funding instruments on the local, national, and
+international level need to be complemented with funding instruments
+specifically designed for research software development and sustained
+research software maintenance to make research software a first class
+citizen in the research landscape. For example, software projects
+enhancing research and fulfilling the sustainability criteria detailed
+in section  on p.  may be entitled for sustained funding as long as they
+live up to the standards and remain a central component of the research
+landscape.
+
+University computing centers and supercomputing facilities need to
+receive earmarked resources for the support of sustainable software
+development. This funding is necessary to provide continuous integration
+services, a hardware portfolio for development, testing and benchmarking
+software, as well as personnel for training domain researchers in
+software design and the proper usage of the services.
+
+The creation and maintenance of training material for general research
+software engineering education and the software-specific documentation
+and tutorial creation needs to be reflected in funding opportunities.
+This can either happen by dedicating modules of research or software
+grants to providing support and the generation of training material, or
+by opening funding schemes focusing on interdisciplinary software
+development education. The latter may include research that looks at
+research software development as a process to analyze which measures,
+interactions, and team compositions make research software successful.
+Additionally, funding instruments fostering the formation of research
+software communities have to be established.
+
+Which Infrastructure is Needed to Sustain Research Software?
+============================================================
+
+> As the *hal9k* community grows, so does the need for infrastructure.
+> Kim and her team collaborate with the National Research Software
+> Engineering Consortium to set up *hal9k* on the Consortium’s
+> distributed TigHub instance, and organize world-wide access to it via
+> the NRSEC-AAI. Going forward, the Consortium’s Research Software Hub -
+> a registry and Software Heritage Archive-based long-term repository
```

Comment:
Add footnote (or something similar) with link to https://www.softwareheritage.org

Reply:
Thanks! Pretty sure we have a proper ref to SWH in here somewhere, so we can re-use it here.

Reply:
```suggestion
> a registry and [softwareHeritage.org](softwareHeritage.org) archive-based long-term repository
```

Reply:
```suggestion
> a registry and [Software Heritage Archive](softwareHeritage.org)-based long-term repository
```